src/instance_store.ts

```diff
@@ -110,12 +110,19 @@ export interface JVBStatus {
     graceful_shutdown: boolean;
 }
 
+export interface whisperStatus {
```

Comment:
Interface names are first-letter capital by convention to differentiate them from the actual instance with values
```suggestion
export interface WhisperStatus {
```